server/routes/common/commonController.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response } from 'express'
+import ReportAProblemPresenter from './reportAProblemPresenter'
+import ReportAProblemView from './reportAProblemView'
+import ControllerUtils from '../../utils/controllerUtils'
+
+export default class CommonController {
```

Comment:
Can we have a unit test for this class, please?

Reply:
sure, I've added a unit test